include/riak_kv_ts.hrl

```diff
@@ -59,7 +59,14 @@
 
 -record(riak_sql_describe_v1,
         {
-          'DESCRIBE' = <<>>  :: binary()
+          'DESCRIBE'    = <<>>  :: binary()
+        }).
+
+-record(riak_sql_insert_v1,
+        {
+          'INSERT'      = <<>>  :: binary(),
+          fields                :: [{identifier, binary()}],
+          values                :: list(list(term()))
```

Comment:
Interesting! Last time I looked, there was no `list()` in `ldbvalue()`. That type was specifically meant to represent all types valid for passing to/from eleveldb, and has been used as such everywhere since 1.0.

As for `number()` standing for `integer()` and `float()`, it's perfectly fine for the purpose of type checking by dialyzer, and that's all that matters here.